extensions/classifiers/proto/text_classifier.ts

```diff
@@ -0,0 +1,45 @@
+import * as pb_1 from "google-protobuf";
```

Comment:
Why is this stuff (and the files below) in this PR?

Reply:
removed.